Python/bytecodes.c

```diff
@@ -132,24 +132,36 @@ dummy_func(
         inst(NOP, (--)) {
         }
 
+        family(RESUME, 0) = {
+            RESUME_CHECK,
+        };
+
         inst(RESUME, (--)) {
+            TIER_ONE_ONLY
```

Comment:
What does this mean? RESUME_CHECK is not TIER_ONE_ONLY and it can depot to this.

Reply:
It means that tier 2 shouldn't attempt to execute it, but exit the region.